src/components/onboarding/follow-map/follow-map-component.scss

```diff
@@ -0,0 +1,3 @@
+.intro--follow {
```

Comment:
This is tough for me.  We're using the `intro` class across multiple pages and we have BEM flags spread across multiple files.  It kind of goes against BEM since the idea is that you can re-use the code by changing the top level.

Would it be possible to use `enter-email`, `follow-map`, etc rather than default to the `intro` class?

Reply:
Skipping